packages/office-ui-fabric-react/src/components/Checkbox/CheckboxPage.visualtest.tsx

```diff
@@ -0,0 +1,15 @@
+import { Checkbox } from './index';
+/* tslint:disable:no-unused-variable */
+import * as React from 'react';
+/* tslint:enable:no-unused-variable */
+export default class CheckboxVPage extends React.Component<any, any> {
+  public render() {
+    return <div >
+      <Checkbox className='Checkbox' label='Check Box'
+        defaultChecked={ true } />
+      <Checkbox className='CheckboxDisbaled' label='Check Box Disabled'
```

Comment:
Updated the code.
Failure case was there in the console log. Missed it in the long list of messages..